granary/templates/index.html

```diff
@@ -138,6 +138,15 @@ <h1>
 </div>
 </div>
 
+<div class="row">
```

Comment:
? odd merge?

Reply:
yeah, i don't know what's up with that. hopefully not a problem?

Reply:
i've rebased on master and this is still here.  any idea how to fix it?